pkg/translate/file.go

```diff
@@ -0,0 +1,118 @@
+package translate
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	pb "go.expect.digital/translate/pkg/server/translate/v1"
+	"golang.org/x/text/language"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+type (
+	uploadTranslationFileRequest   pb.UploadTranslationFileRequest
+	downloadTranslationFileRequest pb.DownloadTranslationFileRequest
+)
```

Comment:
I chose to define new types so that we can create methods for them. 
This approach eliminates the need to create a new function, such as `parseUploadParams` or `parseDownloadParams`, for parsing the parameters for each endpoint. 
Instead, we can parse the parameters using the `.parseParams()` method, without specifying which request we are parsing for explicitly.